src/middlewares/validations.js

```diff
@@ -34,4 +40,30 @@ const validateSignup = user => {
   });
 };
 
-export { validateSignup };
+/**
+ * @param {Request} req Node/Express Request object
+ * @param {Response} res Node/Express Response object
+ * @param {NextFunction} next Node/Express Next callback function
+ * @returns {NextFunction | Object} Node/Express Next callback function or an error response
+* @param {object} oneWayTripRequest
+* @returns {object} verifies one way trip request info
+*/
+const validateOneWayTripRequest = async (req, res, next) => {
+  try {
+    const schema = Joi.object({
+      travelFrom: Joi.string().required(),
+      travelTo: Joi.string().required(),
+      travelDate: Joi.date().required(),
+      travelReason: Joi.string().required(),
+      accommodate: Joi.bool().required(),
+      travelDirection: Joi.equal('one-way').required(),
```

Comment:
when I add spaces at the beginning or at the ending it is refusing the input, same case when I write One-way, and it is a bug.
All these required fields should have specific error messages they throw when not provided.